resources/js/screens/User.vue

```diff
@@ -15,6 +15,26 @@
                         </div>
                     </div>
                 </div>
+                <div class="card mb-3">
+                    <div class="card-body rounded">
+                        <p class="font-weight-bold text-dark text-sm mb-1">Roles</p>
+                        <div v-for="(role, index) in roles" class="d-flex align-items-center">
+                            <div class="custom-control custom-radio">
+                                <input type="radio"
+                                       :id="`role-${index}`"
+                                       name="role-select"
+                                       class="custom-control-input"
+                                       :value="role.id"
```

Comment:
Vue handles strings better than integers in v-model because it uses string comparison `===`
```suggestion
                                       :value="`${role.id}`"
```